### DISCUSSION TITLE
Selenium IDE type command doesn't update x-model variable

### DISCUSSION THREAD

**Discussion by mfwre** [alpinejs/alpine#3903] [1. Help]
I was trying to set up some automated tests using the Selenium IDE extension for Firefox.

I have a form in which, when a value (bound to a text input via the x-model directive) changes triggers some x-if templates.

Having set up and executed the Selenium test when inspected via the AlpineJS dev tools show that the inserted text wasn't getting synced to the bound x-model value.

Clicking and typing on the input test works fine:
![20231206_12h01m34s_grim](https://github.com/alpinejs/alpine/assets/112066022/4be80f2c-9d2c-4980-a1fb-85f6e815d033)

The value doesn't change (modified_quantity should be 202):
![20231206_12h02m35s_grim](https://github.com/alpinejs/alpine/assets/112066022/09f94162-b5c6-4770-88f0-066aa204013b)

**Comment by SimoTod** [comment]
Isn't this a Selenium bug? x-model updates when the browser triggers the standard input event (or the change event for checkbox/radio/select and when you use the lazy modifier). If selenium doesn't replicate what a real user would do when typing, that's not going to work.
